frontend/src/util/attribution.ts

```diff
@@ -7,7 +7,10 @@ export const setAttributionData = () => {
 	const originalReferrer = Cookies.get('referrer')
 
 	if (referrer && !originalReferrer) {
-		Cookies.set('referrer', referrer, { domain: 'highlight.io' })
+		Cookies.set('referrer', referrer, {
+			domain: 'highlight.io',
+			expires: 365,
```

Comment:
i think this value needs to be unix millis? see other places we use `Cookies.set`. so something like
```
        const expires = new Date()
        expires.setTime(expires.getTime() + 15 * 60 * 1000)
        Cookies.set('referrer', referrer, {
	    domain: 'highlight.io',
	    expires,
        })
```

Reply:
@Vadman97 according to the docs, I think it's days from time of creation until it expires:

> Value must be a [Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) which will be interpreted as days from time of creation or a [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) instance.